SPEC_MAINTAINER_PROTOCOL.md

```diff
@@ -0,0 +1,80 @@
+
+Each spec should be maintained at least by 1 person and up to 3 for efficiency purposes.
+
+The spec maintainer(s) are recognized at the top of the specification document on the list "Maintainers(s)" and these are the people that should be requested to review a PR to that specific spec.
+
+### Responsibilities
+
+As a Spec Maintainer, you are expected to:
+
+- Respect and follow the [IPFS Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+- Have a great understanding of the subsystem  purpose and how it is used by other parts of the project.
+- Review and merge PRs to the Spec
+- Respond in a timely manner to Github issues on ipfs/specs repo that are related to the spec being maintained.
+- Notify the developers of the subsystem in case there is a new proposal for a change or also request the developers to notify when there is a system change that needs to be reflected on the spec.
+
+### Specification Status
+
+The Specification Status description can be found on the main [README of this repo](https://github.com/ipfs/specs#badges-and-spec-lifecycle).
+
+## Disclaimer as 2019 Q3
+
+As of 2019 Q3, the specs are vastly out of date and should not be relied uppon. We will run an effort to get these up to date and check the boxes here when it is done:
+
+- **IPFS Protocol:**
+  - [ ] [Protocol Architecture Overview](./ARCHITECTURE.md) - the top-level spec and the stack
+  - [ ] [Other IPFS Overviews](/overviews) - quick overviews of the various parts of IPFS
+- **User Interface (aka Public APIs):**
+  - [ ] [Core API (aka using IPFS as a package/module)](./API_CORE.md)
+    - [x] [JavaScript implementation details](https://github.com/ipfs/interface-js-ipfs-core)
+    - [ ] [Golang implementation details](https://github.com/ipfs/interface-go-ipfs-core)
+  - [ ] [CLI (the ipfs daemon API)](./API_CLI.md)
+  - [ ] [HTTP API](./API_HTTP.md)
+  - HTTP Gateway
+- **Data Formats:**
+  - [ ] [IPLD](https://github.com/ipld/spec) - InterPlanetary Linked Data.
+  - [x] [Merkle DAG (Deprecated)](./MERKLE_DAG.md)
+  - [x] Self Describing Formats ([multiformats](http://github.com/multiformats/multiformats)):
+    - [x] [multihash](https://github.com/multiformats/multihash) - self-describing hash digest format.
+    - [x] [multiaddr](https://github.com/multiformats/multiaddr) - self-describing addressing format.
+    - [x] [multicodec](https://github.com/multiformats/multicodec) - self-describing protocol/encoding streams (note: a file is a stream).
+    - [x] [multistream](https://github.com/multiformats/multistream) - multistream is a format -- or simple protocol -- for disambiguating, and layering streams. It is extremely simple.
```

Comment:
See https://github.com/multiformats/multistream/pull/10. The history around multistream, multicodec, and multistream select are all really confusing and I'm not sure if we can reasonably say that the specs are "up to date".